CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 0.3.0 (2017-xx-xx)
+
+ - ...
```

Comment:
We'll need to add to the changelog, the version will be bumped to `0.3.0`. No need to do this until the end of this change.